V5.3.39 (#7)
YunoHost-Apps/invoiceninja5_ynh: doc/DISCLAIMER.md

### Warning

This is the new version of InvoiceNinja. There are several things not working at the moment:

> * migration from v4

Reviewer: ```suggestion
```

This works now.

* installation into a sub-directory
* sending mails

> Invoice Ninja is the #1 open-source platform to create & email invoices, track payments and expenses, and time billable tasks & projects for clients.

Reviewer: ```suggestion
Invoice Ninja is the the leading self-hosted invoicing platform to create & email invoices, track payments and expenses, and time billable tasks & projects for clients.
```

## YunoHost specific features

#### Multi-user support

* There is no official LDAP support

> * The first user gets created after installing Invoice Ninja

Reviewer: ```suggestion
* The first user gets created after installing Invoice Ninja. The email-address is the one of your administrator user of the app. The password is the administration password you provided while installing.
```
